model/itemTab.cc

```diff
@@ -254,7 +260,7 @@ namespace minsky
                           cairo_set_source_rgba(cairo,0,0,0,0.2);
                           for (y=y0+y1; y<h+rowHeight; y+=2*rowHeight)
                             {
-                              cairo_rectangle(cairo,x0,y,w+colWidth,rowHeight);
+                              cairo_rectangle(cairo,x0,y,w+colWidths.back(),rowHeight);
```

Comment:
I'm little puzzled why the last column width would be appropriate for all columns here...

Reply:
I've generalised the colWidth=std::max(colWidth,pango.width()+5) approach you used so that it is done for every column individually. Now, the width of a given column on the tab will be the maximum pango.width()+5 in that column and not the global maximum in all columns.